### ISSUE TITLE
Followers-only chat?

### ISSUE THREAD

**Issue by ghost** [owncast/owncast#3428]
### Share your bug report, feature request, or comment.

Is there a way to limit chat to followers?

Combined with a private fedi account, this might be a good way to limit  spam for small streamers who might not be able to moderate their chat as actively.

**Comment by gabek** [comment]
There is not. The closest thing is "established user mode", that doesn't allow brand-new users to send to chat. In order to know who is a follower in chat, they would also have to be required to use FediAuth, and authenticate to chat. While some people use that, it's not a heavily used piece of functionality, so I'd be hesitant to build additional features on top of it, requiring users to use it.

It's not a bad idea, but auth, in general, is pretty niche.

**Comment by ghost** [comment]
> they would also have to be required to use FediAuth, and authenticate to chat

Yep, that's what I meant! This is very desirable to me and a few friends I've spoken to.

Not promising that I have the bandwidth at the moment, but just to be sure, would you consider merging PRs for this feature if they came along?

**Comment by gabek** [comment]
Sure! I see this the same way I see chat auth in general. If nobody uses it, it doesn't really get in the way or make the rest of the experience more confusing. It's not something that would have to be "setup" or have lots of configuration that the admin would need to figure out.

**Comment by gabek** [comment]
I just filed https://github.com/owncast/owncast/issues/3482 to discuss some high level changes that we might want to think about if we're going to go in the direction of features that link chat users and Followers.

**Comment by gabek** [comment]
Followers-only chat now exists.
